pages/phases/getting-started/identifying-appropriate-positions.md

```diff
@@ -22,10 +22,14 @@ Your position description (PD) should reflect adequate specialized experience to
 
 ### Identify Billets
 
-To be efficient with SME time, we recommend having at least three and ideally five or more opening/billets to fill in your hiring action. 
+We recommend having at least three and ideally five or more opening/billets to fill in your hiring action. This ensures that the time SMEs spend effectively assessing applicants is leveraged through multiple hires. It also saves time for applicants and HR: if you're able to keep a certiciate open for a year and make 5+ hires off of one hiring action there is less of a need to have applicants apply to many different one-off vacancies.  Ideas if you do not have at least 3-5 openings:
+
+- Try a shared hiring agency with another office or agency with openings for the same position. This way there are more SMEs involved therefore decreasing the overall time burden on one agency's SMEs. USDS recently completed a SME-QA hiring action with 20 SMEs from 9 agencies hiring Customer Experience Specialists. OPM was hired as the HR Specialist and agencies were able to share the certificate the same day. If two agencies want to partner on their own, they are allowed to share the certificate as long as they have applicants opt-in when they apply and have a merit promotion announcement open for 10 days before the second agency makes selections. 
```

Comment:
```suggestion
- Try working with other offices or agencies with openings for the same position. This can also involve more SMEs and decrease the overall time burden on any one group of SMEs. USDS recently completed a SME-QA hiring action with 20 SMEs from 9 agencies hiring Customer Experience Specialists. OPM was hired as the HR Specialist and agencies were able to share the certificate the same day. If two agencies want to partner on their own, they are allowed to share the certificate as long as they have applicants opt-in when they apply and have a merit promotion announcement open for 10 days before the second agency makes selections. 
```